framework/rest/OptionsAction.php

```diff
@@ -40,5 +40,6 @@ public function run($id = null)
         }
         $options = $id === null ? $this->collectionOptions : $this->resourceOptions;
         Yii::$app->getResponse()->getHeaders()->set('Allow', implode(', ', $options));
+        Yii::$app->getResponse()->getHeaders()->set('Access-Control-Allow-Method', implode(', ', $options));
```

Comment:
Should be plural: Methods instead of Method

Reply:
@antonkot good catch. Would you like to submit a PR with fix?

Reply:
sure

Reply:
@SilverFire Here it is: #15869